### DISCUSSION TITLE
ol, ul and blockquotes have no decoration

### DISCUSSION THREAD

**Discussion by technologyconsulting** [remarkjs/remark#681] [Q&A] [answered]
I'm probably misunderstanding something but the following have no decoration. What I mean is an ol has no numbers, ul has no dashes and blockquotes have no pipes.

However, the JSX seems to be correct. Should I expect them to be visible?

Thanks

Simon

**Comment by ChristianMurphy** [comment, marked as answer]
I'm not sure which package specifically you are referring to, maybe [`react-markdown`](https://github.com/remarkjs/react-markdown)?
`react-markdown` does not include styles, allowing the adopters/implementers to choose the appearance.

> What I mean is an ol has no numbers, ul has no dashes and blockquotes have no pipes.

Are you using tailwind css by any chance?
This is a design decision by tailwind https://github.com/remarkjs/react-markdown/issues/505, several folks have gone with tailwind `typography` + `prose` as a solution https://github.com/remarkjs/react-markdown/issues/505#issuecomment-724911501

Or if you are looking for something that looks more like GitHub, consider adding https://github.com/sindresorhus/github-markdown-css to the stylesheet

**Reply by technologyconsulting** [reply]
Hi Christian,

Apologies,  I thought I had opened up a discussion on the React Markdown page.

Tailwind css, yes I am.

Thanks for the insight and pointers Christian. I'll look over the options in the morning. 

Simon

**Reply by ChristianMurphy** [reply]
>  I thought I had opened up a discussion on the React Markdown page.

All good, we centralize discussions in one place, on the `remark` repo, which helps discussions to be easier to find, but does mean extra clarification is needed sometimes.
